stylesheets/highlight.scss

```diff
@@ -12,9 +12,7 @@ pre code {
 }
 
 pre .comment,
-pre .template_comment,
-pre .diff .header,
-pre .javadoc {
+pre .err {
```

Comment:
Good idea! I'll update this now. I'm also gonna merge the syntax highlighting rules into the main compiled css, since there's no point in paying an extra HTTP request for a few rules.